genie-web/src/test/java/com/netflix/genie/web/services/impl/JobKillServiceV3UnitTests.java

```diff
@@ -49,28 +49,29 @@
 import java.util.UUID;
 
 /**
- * Unit tests for the LocalJobKillServiceImpl class.
+ * Unit tests for the JobKillServiceV3 class.
  *
  * @author tgianos
  * @since 3.0.0
  */
 @Category(UnitTest.class)
-public class LocalJobKillServiceImplUnitTests {
+public class JobKillServiceV3UnitTests {
 
     private static final String ID = UUID.randomUUID().toString();
-    private static final String HOSTNAME = UUID.randomUUID().toString();
+    private static final String hostName = UUID.randomUUID().toString();
```

Comment:
This is a constant, i'm surprised checkstyle doesn't complain about it being lowercase..